fmn/consumer/producer.py

```diff
@@ -114,7 +114,8 @@ def manage_batch(session, backend, pref):
                 backend.handle(session, recipient, msg, streamline=True)
             else:
                 # Otherwise, send it as a batch/digest
-                backend.handle_batch(session, recipient, queued_messages)
+                backend.handle_batch(session, recipient,
+                                     [q.msg for q in queued_messages])
```

Comment:
It seems like you have changed this API subtly. Wouldn't this warrant a X version raise and a release note? Or is this API only used locally within this codebase?